src/scss/skin-modern/components/_button.scss

```diff
@@ -6,12 +6,13 @@
   background-color: transparent;
   background-position: center;
   background-repeat: no-repeat;
+  background-size: 1.5em;
   border: 0;
   cursor: pointer;
   font-size: 1em;
   height: 1.5em;
   min-width: 1.5em;
-  padding: 0;
+  padding: 1em;
```

Comment:
Why did you take this approach now, instead of 

```
background-origin: content-box;
padding: .25em;
box-sizing: content-box;
```
like in #178?

Reply:
Because it's not needed when setting the background-size. (And it's not working without more changes)

Reply:
This might not work with irregular paddings, e.g. a larger padding on the bottom of a button to fill unused space... the background image will always be centered, `background-origin: content-box` seems not to work, and the background needs to be explicitely positioned with `background-position: x y`.

Reply:
That was the case for `box-sizing: content-box !important;`. Then `background-origin` will work. So let's merge #181 first, then i can change it.

Reply:
Merged.